src/Pusher.php

```diff
@@ -745,4 +751,48 @@ public function notify($interests, $data = array(), $debug = false)
 
         return false;
     }
+
+    /**
+     * Verify that a webhook actually came from Pusher, and marshals them into a PusherWebhook object.
+     *
+     * @param array  $headers a array of headers from the request (for example, from getallheaders())
+     * @param string $body    the body of the request (for example, from file_get_contents('php://input'))
+     *
+     * @return PusherWebhook object with the properties time_ms (an int) and events (an array of event objects)
+     */
+    public function parse_webhook($headers, $body)
```

Comment:
To be consistent with our existing libraries (e.g. Ruby, Node, Go) I think this method should be called `webhook` and the returned object should be called `Webhook`.

Reply:
I think `Webhook` is appropriate since we're in the `Pusher` namespace anyway.

Reply:
Fair enough